SPECS/grub2/grub2.spec

```diff
@@ -46,6 +46,8 @@ Patch0156:      0156-efilinux-Fix-integer-overflows-in-grub_cmd_initrd.patch
 Patch0157:      0157-linuxefi-fail-kernel-validation-without-shim-protoco.patch
 # Fix to prevent user from overwriting signed grub binary using grub2-install
 Patch0166:      0166-grub-install-disable-support-for-EFI-platforms.patch
+# CVE-2021-3981
+Patch0167:      0167-restore-umask-for-grub-config.patch 
```

Comment:
Asking because I'm unsure-will our CVE tooling detect this fix?

Reply:
Need to double check on this. I'm following what was done for CVE-2020-15705 in this package.

Reply:
Confirmed. The CVE number is mentioned in the patch file, so it will be detected by our CVE tooling.

Reply:
Neat, thanks for checking!